internal/endtoend/testdata/select_system/pgx/sqlc.yaml

```diff
@@ -0,0 +1,14 @@
+version: "2"
+cloud:
+  project: "01HAQMMECEYQYKFJN8MP16QC41"
```

Comment:
We're still replacing this and the `database` config with test context in exec.json no?

Reply:
D'oh, removed